src/oc_erchef/apps/oc_chef_wm/src/oc_chef_wm_sup.erl

```diff
@@ -91,10 +92,11 @@ maybe_start_action(false, Workers) ->
     Workers.
 
 
-check_queue_at_capacity(Vhost, Queue) ->
+check_actions_queue_at_capacity(Vhost, Queue) ->
```

Comment:
[minor] Perhaps pass the pool name since we've already looked it up at the only call site of this function.